src/lib/Microsoft.Fx.Portability.Git/ApiPortGitRepositoryStreamBuilder.cs

```diff
@@ -0,0 +1,356 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using LibGit2Sharp;
+using Microsoft.Fx.Portability.ObjectModel;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Microsoft.Fx.Portability
+{
+    /// <summary>
+    /// Builds <see cref="ApiPortData"/> using a Git repository stream.
+    /// </summary>
+    public class ApiPortGitRepositoryStreamBuilder
```

Comment:
Do we still use this? You're downloading the data directly, right? If so, this probably isn't needed